django_ninja_crudl/endpoints/list.py

```
@@ -1,15 +1,15 @@
~"""CRUDL API base class."""
 
 import logging
 from abc import ABC
+from typing import TYPE_CHECKING, Unpack
 
 from django.db import models
 from django.db.models import (
     ForeignKey,
+    ManyToOneRel,
```

Ruff (0.8.2)

`django.db.models.ManyToOneRel` imported but unused

Remove unused import

(F401)